src/playlist-loader.js

```diff
@@ -395,12 +395,7 @@ export default class PlaylistLoader extends EventTarget {
 
     this.customTagParsers = (vhsOptions && vhsOptions.customTagParsers) || [];
     this.customTagMappers = (vhsOptions && vhsOptions.customTagMappers) || [];
-    this.llhls = (vhsOptions && vhsOptions.llhls) || false;
-
-    // force llhls for IE 11
-    if (videojs.browser.IE_VERSION) {
-      this.llhls = false;
-    }
+    this.llhls = (vhsOptions && vhsOptions.llhls) || true;
```

Comment:
This line will always evaluate to true. But I think the option should to be set at a higher level (in src/videojs-http-streaming.js, probably in `setOptions_` https://github.com/videojs/http-streaming/blob/main/src/videojs-http-streaming.js#L622 ).